airflow/dag_processing/bundles/base.py

```diff
@@ -103,6 +108,8 @@ def view_url(self, version: str | None = None) -> str | None:
         """
         URL to view the bundle.
```

Comment:
what does it mean to "view the bundle"--- where? in airlfow ui?  or some external website?